ext/mbstring/tests/mb_str_split.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+mb_str_split()
+--SKIPIF--
+<?php extension_loaded('mbstring') or die('skip mbstring not available'); ?>
+--INI--
+output_handler=
+mbstring.func_overload=0
+--FILE--
+<?php
+ini_set('include_path','.');
+include_once('common.inc');
+
+// Japanese
+$utf8_jp = "どうも";
+$euc_jp = mb_convert_encoding($utf8_jp, "EUC-JP", "UTF-8");
+
+// Russian
+$utf8_ru_1 = "Миру";
+$utf8_ru_2 = "Мир";
+$cp866_ru = mb_convert_encoding($utf8_ru_2, "CP866", "UTF-8");
+
+
+printf("1: %s\n", serialize(mb_str_split($utf8_ru_1, 1 , "UTF-8")));
+printf("2: %s\n", mb_convert_encoding(serialize(mb_str_split($cp866_ru, 1 , "CP866")), "UTF-8", "CP866"));
+printf("3: %s\n", mb_convert_encoding(serialize(mb_str_split($euc_jp, 1, "EUC-JP")), "UTF-8", "EUC-JP"));
+printf("4: %s\n", mb_convert_encoding(serialize(mb_str_split($euc_jp, 3, "EUC-JP")), "UTF-8", "EUC-JP"));
```

Comment:
I think it would make sense to add some more tests here for completness.
E.g.:
1. UTF-16 text that uses surrogate pairs (make sure that we're treating the pairs as a single character)
2. UTF-8 (or other UTF) text that uses Combining Characters (if I understand everything correctly, those would be treated as separate characters, which would be consistent with the rest of mbstring extension, but in my opinion a bit weird, so I think it would be good to have a test so that behaviour is explicit).